File: TourismAgency/src/main/java/domain/TravelAgent.java
package domain;

public class TravelAgent extends Entity<Long>{
    private String username;
    private String passwd;

    public TravelAgent(String username, String passwd) {
        this.username = username;
        this.passwd = passwd;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPasswd() {
        return passwd;
    }

    public void setPasswd(String passwd) {
        this.passwd = passwd;
    }

    @Override
    public String toString() {
        return "TravelAgent{" +
                "username='" + username + '\'' +
                ", passwd='" + passwd + '\'' +
                '}';
    }
}

File: AgentieTurism/model/domain/TravelAgent.cs


using System;

namespace model.domain
{
    [Serializable]
    public class TravelAgent :Entity<long>
    {
        public string Username { get; set; }
        public string Passwd { get; set; }

        public TravelAgent(string username, string passwd)
        {
            Username = username;
            Passwd = passwd;
        }

        public override string ToString()
        {
            return "Username: "+Username+" Password: "+Passwd;
        }

       
    }
}
